src/System.Private.Xml/tests/XmlSerializer/XmlSerializerTests.netcoreapp.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Xml;
+using System.Xml.Serialization;
+using Xunit;
+
+public static partial class XmlSerializerTests
+{
+    private static readonly string SerializationModeSetterName = "set_Mode";
+
+    static XmlSerializerTests()
+    {
+        var method = typeof(XmlSerializer).GetMethod(SerializationModeSetterName, BindingFlags.NonPublic | BindingFlags.Static);
```

Comment:
I just copied it over from the original file, but since I'm already on this I can update it :)